disnake/audit_logs.py

```diff
@@ -676,8 +676,8 @@ def _from_data(self, data: AuditLogEntryPayload) -> None:
         self.user = self._get_member(utils._get_as_snowflake(data, "user_id"))  # type: ignore
         self._target_id = utils._get_as_snowflake(data, "target_id")
 
-    def _get_member(self, user_id: int) -> Union[Member, User, None]:
-        return self.guild.get_member(user_id) or self._users.get(user_id)
+    def _get_member(self, user_id: int) -> Union[Member, User, Object]:
+        return self.guild.get_member(user_id) or self._users.get(user_id) or Object(id=user_id)
```

Comment:
```suggestion
    def _get_member(self, user_id: int) -> Union[Member, User, Object, None]:
        if not user_id:
            return
        return self.guild.get_member(user_id) or self._users.get(user_id) or Object(id=user_id)
```
Contrary to what the annotations indicate, `user_id` can be `None` in some cases, which would break `Object()` :/